pages/blog.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Blog from '../components/Blog'
+
+function BlogPage () {
+  return (
+    <Blog />
```

Comment:
Let's not nest where nesting doesn't provide any benefit. The markup and logic in `<Blog />` can be moved to this page.

Reply:
`Blog` component requires to maintain some state logic such as `const [blogs, blogSchools, selectedSchool, updateSelectedSchool] = useToRenderBlog()` and useToRenderBlog internally maintains few states like `selectedSchool`, `updateSelectedSchool` and so on.. 

I modularized the code base hence I think moving entier `blogPage.js` is not a good idea... 

But if API integration is removed and uses static JSON to represent data might I can be able to move `blogPage.js` in this file still there will be an issue as discussed in `https://github.com/teacherfund/TeacherFund_next/pull/145#discussion_r333737352`

But still creating blog page out an independent component would be cluttered...

Pls drop your comments...